src/lib/constants/defaultModelObjects.ts

```diff
@@ -10,3 +10,30 @@ export const DEFAULT_CATEGORY_COMBO = {
     isDefault: true,
     name: 'default',
 }
+
+export const DEFAULT_CATEGORY = {
+    id: 'GLevLNI9wkl',
+    isDefault: true,
+    name: 'default',
+}
+
+export const DEFAULT_CATEGORYCOMBO_SELECT_OPTION = {
```

Comment:
I'm not wild about hardcoding all of these default items, but this might be something we want to check with the backend team about. These are hardcoded on the backend, and since this app has min version 2.41, I think it would be probably impossible to still have legacy non-standard defaults floating around if you're using v41.